recipes/xdg/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "xdg" %}
+{% set version = "1.0.5" %}
+{% set sha256 = "d495c8c620a6a34b6111f3254d6fab5f06c1a7ce532e8f34e029209a19e6ce65" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: True  # [win]
```

Comment:
Not sure if it makes sense to build this package on windows also, but it can not hurt right?

Reply:
The package won't produce correct results on windows right now. Since some of the environment variables the library requests are not defined on windows it's likely that an error is thrown. So not building on windows is the better alternative.

Reply:
Ok. Thanks for the explanation.